TurkishFilter/sections/specific.txt

```diff
@@ -45,6 +45,8 @@ megahaber27.com,yemektarifleri.org,lifebursa.com,mshowto.org,macizlee1.com,maciz
 ! ###adStarter
 fullhdfilmcibaba.com,720pizlemek.com,hdfilmizledayi.com,fullhdfilmizlebaba.com###adStarter
 !
+istikbalgazetesi.com##div[align="center"] > a[target="_blank"] > img
+||istikbalgazetesi.com/gurlife.jpg
```

Comment:
please remove this rule and add:
`istikbalgazetesi.com##.side-banner`

Reply:
Should I remove `||istikbalgazetesi.com/gurlife.jpg` ?

Reply:
@gork7777 yes